src/engine/Algebra.ts

```diff
@@ -68,6 +68,15 @@ export class Vector {
       return true;
    }
 
+  /**
+  * Calculates distance between two Vectors
+  * @param vec1
+  * @param vec2
+  */
+  public static distanceBetweenVectors(vec1: Vector, vec2: Vector) {
```

Comment:
Ah yes the indentation was off, thanks. 

Will rename as well, originally was unsure how to name as the instance method is also called distance and I wanted to avoid confusion. But I guess that is okay or even preferable?